biohansel/cli.py

```diff
@@ -247,26 +253,56 @@ def parse_comma_delimited_floats(ctx: click.Context, param: click.Option, value:
 
 
 @cli.command()
-@click.option('--vcf-file-path',
+@click.option('-v', '--vcf-file-path',
               required=True,
               type=click.Path(exists=True, dir_okay=False),
               help='Variant calling file (VCF) of a collection of input genomes for population of interest against a '
                    'reference genome that must be specified with --reference-genome-path')
-@click.option('--reference-genome-path',
+@click.option('-r', '--reference-genome-path',
               required=True,
               type=click.Path(exists=True, dir_okay=False),
               help='Reference genome assembly file path. The reference used in the creation of the input VCF file.')
 @click.option('--phylo-tree-path',
               required=False,
               type=click.Path(exists=True, dir_okay=False),
               help='Optional phylogenetic tree created from variant calling analysis.')
-@click.option('--distance-thresholds',
+@click.option('-d', '--distance-thresholds',
               required=False,
               type=str,
               callback=parse_comma_delimited_floats,
               help='Comma delimited list of distance thresholds for creating hierarchical clustering groups '
                    '(e.g. "0,0.05,0.1,0.15")')
-def create(vcf_file_path, reference_genome_path, phylo_tree_path, distance_thresholds):
+@click.option('-o', '--output-folder-name',
+              required=True,
+              type=click.Path(exists=False, dir_okay=True),
+              help='Output folder name in which schema file would be located'
+              )
+@click.option('-s', '--schema-name',
+              required=False,
+              type=str,
+              help='A unique name for the schema file that is generated, the default is just'
+                   '{bio_hansel-schema-reference_genome_name}-{schema_version}')
+@click.option('-m', '--schema-version',
+              required=False,
+              type=str,
+              help='An optional version number for the schema file that is generated')
+@click.option('-p', '--padding-sequence-length',
+              required=True,
+              type=int,
+              help='Output folder name in which schema file would be located'
+              )
+@click.option('-f', '--reference-genome-format',
```

Comment:
yes this has been addressed in commit 0175d925d1a6a25cbd9b292422c36869fe85914f, I kept this variable since the genbank format definitely comes up alot as well. Also Biopython has a built in fasta parser, so that was the one I wanted to keep using. Let me know if that's ok

Reply:
Sure, that's fine. However, please make this option optional (i.e. `required=False`) and try to find out the format either from the filename or try parsing using either format's BioPython parser. Let's try to let users run `biohansel create ...` while specifying as few options as possible where the defaults make sense given their input data. 

If a GenBank format file is supplied and it has annotations (e.g. Prokka output or RefSeq GenBank file), it may be useful to output what the markers target in a table (e.g. marker at position 123 in contig *abc* in genome *X* lies in CDS with gene name *xyz* and product "some protein"). This could be useful information for understanding the scheme output.

Reply:
yes this has been addressed in commit b554b060efeb93d1eac0414a82f84a6088ac8b6c where the user has the option to specify the format, but is not required to